site/docs/about/research.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: Our approach to research
+layout: DetailTechnical
+sidebar:
+  label: Research approach
+---
+
+The Salt Design System team puts people at the heart of its design process. This philosophy is most evident in the team’s user-centered approach to usability research.
+
+Research is vital to our efforts to build a fast, modern and open design system. It means we can collect data about how our system works in practice, then use that data to improve the system and create a better product for users.
+
+When it comes to research, our key principles are:
+
+- Mix methods: we adopt a holistic approach that includes qualitative and quantitative data to best serve evidence-based recommendations to our builders and internal decision makers
+- Build to scale: we aim to make it easier to reuse our methods and report results
+
+The users of our design system are:
+
+- Product teams: developers, designers, content teams and beyond all use our design system to create experiences in their applications
+- End-user: individuals or teams that use applications built with Salt
+
+We consider how the designer and developer interpret the design system and how they adjust it for the end user. The usability of an application is dependent on two parties: our team and the application team using the design system.
+
+<img
```

Comment:
Can we use the `<Image />` component instead? It should also help with responsiveness

Reply:
I did originally, but it ignores the `style` prop, so I wasn't able to add a margin below it. TBH, the out-of-the-box styling probably should have suitable margins between any block-level items on the page. I'll take another look to see if I can fix this without breaking the formatting on other pages.

Reply:
Do we definitely need that margin? We don't seem to be adding in on other pages

Reply:
It looks a bit unbalanced without. The image itself already has some padding within it, but it's equal amounts top & bottom.

It seems our CSS applies a margin bottom to paragraphs, so there's a margin between the paragraph above the image and the upper edge of the image. But there's no margin between the bottom edge of the image and the paragraph below.

If this was a photo (or any image with a non-white background), it would be much more obvious.

Reply:
Makes sense, in that case we should probably fix it across all images (which we could do on a separate PR) although it would still be good to fix the responsiveness

Reply:
I looked into it and the trouble is that the `<Image />` component is one of the Mosaic ones, and I don't see a way to either modify it or target it in our CSS via its class name (which I think must be generated by their setup - Vanilla Extract, or whatever it is).

So, as an interim (and because they may come in handy in future), I've added some CSS utility classes and used that to add the margin below for now.

We've just encountered a similar issue with a missing gap below a table on the duration page, so I'll raise a new GitHub issue to revisit our Markdown stlying & components holistically and see if we can find a more robust solution for these sorts of things.

Reply:
sounds good!